stories/topics/proc-gcp-generate-playbook.adoc

```diff
@@ -0,0 +1,48 @@
+[id="proc-gcp-generate-playbook"]
+
+= Generating the playbook
+
+To generate the playbook, run the command generator to generate the CLI command.
+
+[source,bash]
+----
+docker run --rm -v /tmp:/data $IMAGE command_generator gcp_setup_logging_monitoring \
+--data-file /data/logging-monitoring.yaml
+----
```

Comment:
Don't we need to show the data-file contents so the user understands how the next command data is generated with the data it has?

Reply:
Ignore this comment.  Now that I see the entire html doc, I can see that this is not an issue.